### DISCUSSION TITLE
Help for ingame help needed

### DISCUSSION THREAD

**Discussion by webfischi** [Keriew/augustus#256] [Q&A]
Augustus has a lot of new game mechanics, that should be explained in the in game help. It would be great, if someone who understands this mechnic could write a help text, explaining the mechanics. Either directly into the code as a pull request or here in this discussion.

**Comment by Riww** [comment]
I think for start, at least the menu Help-About shall say that this Augustus build, version number, and possibly people involved.

There is pdf manual which comes with Augustus, so maybe the texts there could be used for ingame help.

**Reply by crudelios** [reply]
> I think for start, at least the menu Help-About shall say that this Augustus build, version number, and possibly people involved.

Humm, maybe an "About Augustus" in addition to "About"!

> There is pdf manual which comes with Augustus, so maybe the texts there could be used for ingame help.

That's our idea actually, write the texts so they can be used on both places interchangeably with minimal effort 👍
